source/_components/arcam_fmj.markdown

```diff
@@ -0,0 +1,102 @@
+---
+layout: page
+title: "Arcam FMJ Receivers"
+description: "Instructions on how to integrate Arcam FMJ Receivers into Home Assistant."
+date: 2019-04-28 13:59 +0200
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: arcam.svg
+ha_category: Media Player
+ha_release: 0.96
+---
+
+The `arcam_fmj` integration allows you to control [Arcam FMJ Receveivers](https://www.arcam.co.uk/range/fmj.htm) from Home Assistant.
+
+Supported devices:
+
+- AVR 380 
+- AVR 450
+- AVR 750
+- Likely other AVRs
+
+## {% linkable_title Configuration %}
```

Comment:
We shouldn't use `linkable title` anymore, correct @frenck? Just `## Configuration`

Reply:
Thats right, it's no longer needed. But it doesn't break anything if you let it in, it continues to work 😉

Reply:
Thanks for clarification! :+1:

Reply:
I'll correct the pull before merge. I still have a day or two :)